fn_uptime_data_generator/handler.py

```
@@ -20,8 +20,11 @@
 def handle(data: dict, client: CogniteClient) -> dict:
     today = pd.Timestamp(dt.datetime.today().strftime("%Y-%m-%d"))
 
-    insert_uptime_kpi(today, client)
-    return {"status": "succeeded", "data": data}
+    try:
+        insert_uptime_kpi(today, client)
+        return {"status": "succeeded", "date": today.isoformat()}
+    except Exception as e:
+        return {"status": "failed", "date": today.isoformat(), "cause": e}
```

you have try except in function insert_uptime_kpi which will catch all Exceptions so this one will not catch anything.
Maybe yo need to narrow down errors one level deeper or "raise" after logging by sentry